tests/tracer/test_propagation.py

```diff
@@ -20,19 +20,240 @@
 from ddtrace.propagation.http import _HTTP_HEADER_B3_SINGLE
 from ddtrace.propagation.http import _HTTP_HEADER_B3_SPAN_ID
 from ddtrace.propagation.http import _HTTP_HEADER_B3_TRACE_ID
+from ddtrace.propagation.http import _HTTP_HEADER_TAGS
 
 from ..utils import override_global_config
 
 
 NOT_SET = object()
 
 
+def test_inject(tracer):
+    meta = {"_dd.p.dm": "value", "_dd.p.other": "value", "something": "value"}
+    ctx = Context(trace_id=1234, sampling_priority=2, dd_origin="synthetics", meta=meta)
+    tracer.context_provider.activate(ctx)
+    with tracer.trace("global_root_span") as span:
+        headers = {}
+        HTTPPropagator.inject(span.context, headers)
+
+        assert int(headers[HTTP_HEADER_TRACE_ID]) == span.trace_id
+        assert int(headers[HTTP_HEADER_PARENT_ID]) == span.span_id
+        assert int(headers[HTTP_HEADER_SAMPLING_PRIORITY]) == span.context.sampling_priority
+        assert headers[HTTP_HEADER_ORIGIN] == span.context.dd_origin
+        assert ("_dd.p.dm=value,_dd.p.other=value" in headers[_HTTP_HEADER_TAGS]) or (
```

Comment:
Below you do this, any reason to be different here?

```python
        # The ordering is non-deterministic, so compare as a list of tags
        tags = set(headers[_HTTP_HEADER_TAGS].split(","))
        assert tags == set(["_dd.p.dm=bytes"])
```

I actually don't care that much, curiosity, feel free to resolve this without any action.